addons/account/models/account_move.py

```diff
@@ -3978,6 +3978,14 @@ def _notify_get_recipients_groups(self, msg_vals=None):
     def _get_report_base_filename(self):
         return self._get_move_display_name()
 
+    def _get_report_attachment_filename(self):
+        self.ensure_one()
+        return (self.state == 'posted') and ((self.name or 'INV').replace('/', '_') + '.pdf')
+
+    def _get_report_mail_attachment_filename(self):
+        self.ensure_one()
+        return 'Invoice_' + (self.name or '').replace('/', '_') + (self.state == 'draft' and '_draft' or '')
```

Comment:
`Invoice` should be translate:
```suggestion
        invoice_name = (self.name or '').replace('/', '_')
        draft_suffix =  _('_draft') if self.state == 'draft' else ''
        return _('Invoice_%(name)s%(draft)s', {'name': invoice_name, 'draft': draft_suffix})
```